example_code/basic_examples/abi_decode/Cargo.toml

```diff
@@ -0,0 +1,31 @@
+[package]
+name = "stylus-decode-hashing"
+version = "0.1.7"
+edition = "2021"
+license = "MIT OR Apache-2.0"
+keywords = ["arbitrum", "ethereum", "stylus", "alloy"]
+
+[dependencies]
+alloy-primitives = "0.7.6"
+alloy-sol-types = "0.7.6"
```

Comment:
Think we need to "=0.7.6" to make sure the version is pinned to a specific minor